addon/index.js

```diff
@@ -57,7 +62,8 @@ export default class Resolver {
   }
 
   resolve(fullName) {
-    let moduleName = this.moduleNameForFullName(fullName);
+    const _fullName = this.failSet[fullName] || fullName;
```

Comment:
change related to the above recommendation: `this.failSet && this.failSet[fullName] || fullName;`